R/standalone-check_pkg_installed.R

```diff
@@ -100,12 +100,12 @@ is_pkg_installed <- function(pkg,
 #' @inheritParams check_pkg_installed
 #' @keywords internal
 #' @noRd
-get_pkg_dependencies <- function(reference_pkg = "cards", lib.loc = NULL) {
-  if (rlang::is_empty(reference_pkg)) {
+get_pkg_dependencies <- function(ref = "cards", lib.loc = NULL) {
```

Comment:
Instead of renaming this argument to `ref`. For this function, would it make more sense for the argument to be `pkg` ? What do you think?

Reply:
Yes, I think that makes more sense here - shall I change it to `pkg`?

Reply:
Yes, thank you! 🙇🏼‍♂️

Reply:
Done!